freqtrade/freqtradebot.py

```diff
@@ -100,6 +100,8 @@ def __init__(self, config: Dict[str, Any]) -> None:
         self._exit_lock = Lock()
         LoggingMixin.__init__(self, logger, timeframe_to_seconds(self.strategy.timeframe))
 
+        self.last_process = datetime.utcfromtimestamp(0.0)
```

Comment:
using `utcfromtimestamp` is dangerous, as it generates a timezone-native object.

if you run `datetime.utcfromtimestamp(0.0).timestamp()` - then the result is -3600 (my timezone is +1).